ebb_lint/test/test_flake8.py

```diff
@@ -1,13 +1,15 @@
+# -*- coding: utf-8; -*-
 from __future__ import unicode_literals
 
 import ast
 import functools
+import pytest
 import re
+import six
 import sys
 
-import pytest
-import six
 from flake8.engine import get_parser
+from io import BytesIO
```

Comment:
Ahah. My mental model overlaps with that like 70%, but distinctly short of 100%. Probably best for me to use project conventions for projects that aren't mine. 

(my mental model is more like `__future__` -> any external import that's just `import foo` -> any external import that's `from foo import bar, baz` -> intra-package imports)